README.md

```diff
@@ -7,7 +7,6 @@
   - [🛠 Built With ](#-built-with-)
     - [Key Features ](#key-features-)
     - [Home page](#home-page)
```

Comment:
- [ ] Please there is a border upper the books form make sure to implement it , 
Your design 
![image](https://github.com/jkanyi-web/bookstore-cra/assets/74744787/25b1b264-6101-4112-8de8-22237278b941)
Design 
![image](https://github.com/jkanyi-web/bookstore-cra/assets/74744787/3c75d5b1-c6a7-42f4-9a67-1de2f32ffb26)

Reply:
@ [MeqdamAlqudah] why are you not considerate of my current time situation? Do you want me to fail my week?